pkg/test/test.msbuild

```diff
@@ -4,22 +4,44 @@
     <TestPackages Condition="'$(TestPackages)' != ''" Include="$(TestPackages)" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <PackageTestProjectsDir Condition="'$(PackageTestProjectsDir)' == ''">$(MSBuildThisFileDirectory)..\projects</PackageTestProjectsDir>
+    <_projectProperties>SupportFilesDir=$(MSBuildThisFileDirectory)</_projectProperties>
+
+    <!-- We need this to be able to pass restore sources from command line when running tests in helix -->
+    <_restoreSources Condition="'$(PackageTestSources)' != ''">$([System.String]::Copy('$(PackageTestSources)').Replace('+',';'))</_restoreSources>
+  </PropertyGroup>
+
   <Target Name="GetProjects">
     <ItemGroup>
-      <Project Condition="'@(TestPackages)' == ''" Include="projects\**\*.csproj" />
-      <Project Condition="'@(TestPackages)' != ''" Include="projects\%(TestPackages.Identity)\**\*.csproj" />
+      <Project Condition="'@(TestPackages)' == ''" Include="$(PackageTestProjectsDir)\**\*.csproj" />
+      <Project Condition="'@(TestPackages)' != ''" Include="$(PackageTestProjectsDir)\%(TestPackages.Identity)\**\*.csproj" />
     </ItemGroup>
   </Target>
 
   <Target Name="Restore"
           Inputs="@(Project)"
           Outputs="@(Project->'%(RootDir)%(Directory)obj\project.assets.json')"
           DependsOnTargets="GetProjects">
-    <MSBuild Projects="@(Project)" Targets="Restore" BuildInParallel="true" />
+
+    <ItemGroup>
+      <!-- This Split() is needed since the semicolon in ProcessedTargetQueues is now a literal -->
+      <_normalizedRestoreSources Condition="'$(_restoreSources)' != ''" Include="$(_restoreSources.Split(';'))" />
```

Comment:
What is ProcessedTargetQueues?

Reply:
Forgot to cleanup, I'm going to delete these stuff.